modules/calib3d/test/test_cameracalibration.cpp

```diff
@@ -1346,6 +1346,108 @@ void CV_ProjectPointsTest_CPP::project( const Mat& objectPoints, const Mat& rvec
 
 ///////////////////////////////// Stereo Calibration /////////////////////////////////////
 
+class CV_StereoCalibrationCornerTest : public cvtest::BaseTest
+{
```

Comment:
I follow the "consistency is better than brilliance" principle. Every other test in this file looks like this. So, I follow the common pattern. If the conversion should happen, it should, probably, happen all at once.